docs/src/main/sphinx/language/types.rst

```diff
@@ -239,11 +239,13 @@ The following examples illustrate the behavior::
 Instant in time that includes the date and time of day with ``P`` digits of
 precision for the fraction of seconds and with a time zone. Values of this
 type are rendered using the time zone from the value.
-Time zones are expressed as the numeric UTC offset value::
+Time zones are expressed as the numeric UTC offset value or a full name::
```

Comment:
Should we detail what "full name" means and what standard list is used? Maybe link to it?

Reply:
yes!

ours are IANA zone names. there are other standards of zone naming

Reply:
Ahhh it's IANA! I tried to figure out which acronym consortium was responsible for time zones but conveniently forgot about them. Will add that.